share/sensitivity-analysis/README.md

```diff
@@ -0,0 +1,16 @@
+# Cyclus-Dakota Sensitivity Analysis 
+
+This folder contains a test example of Cyclus sensitivity analysis studies conducted using Dakota. Cyclus interfaces with Dakota through a Python interface.
+
+To run the test case, run the following from this directory:
+```
+dakota -i dakota_test.in -o dakota_test.out
+```
+
+In the test case, dakota is running 2 Cyclus simulations in which power is varied from 1500 to 1600MW with 2 partitions. This results in 3 Cyclus simulations run for 1500MW, 1550MW, and 1600MW. 

```

Comment:
Is it 2 or 3 simulations?  While I don't want a full Dakota tutorial, I am curious to know where in the file it specifies how many cases to run?

Reply:
In `dakota_test.in`, you define the number of partitions (n) you want, and it will run n+1 number of cases from 1500 to 1600MW (uniformly distributed)

Reply:
You have `partitions = 1` so 2 simulations.  Are they at the end points: 1500 & 1600?

Reply:
I forgot to push a change from my branch. Just did! I changed the partition to 2.

Reply:
So that makes it 1500, 1550 and 1600MW